src/ert/gui/simulation/view/realization.py

```diff
@@ -153,3 +169,22 @@ def _paint_inner_grid(painter: QPainter, rect: QRect, colors) -> None:
 
     def sizeHint(self, option, index) -> QSize:
         return self._size
+
+    def eventFilter(self, watched, event: QEvent):
+        if event.type() == QEvent.Type.ToolTip:
+            mouse_pos = event.pos()
+            parent: RealizationWidget = self.parent()
+            view = parent._real_view
+            index = view.indexAt(mouse_pos)
+            if index.isValid():
+                mem_msg = index.data(MemoryUsageRole)
+                (current_memory_usage, maximum_memory_usage) = mem_msg
+                if current_memory_usage and maximum_memory_usage:
+                    txt = (
+                        f"{'Current memory usage:':<24} {byte_with_unit(current_memory_usage)}\n"
+                        f"Maximum memory usage: {byte_with_unit(maximum_memory_usage)}"
+                    )
```

Comment:
```suggestion
                    txt = (
                        f"Current memory usage:\t {byte_with_unit(current_memory_usage)}\n"
                        f"Maximum memory usage:\t {byte_with_unit(maximum_memory_usage)}"
                    )
```

This yields perfect alignment on my setup